inst/traduire/en-translation.json

```diff
@@ -233,6 +233,28 @@
     "PROGRESS_DOWNLOAD_COARSE": "Generating coarse-output download",
     "PROGRESS_DOWNLOAD_SUMMARY": "Generating summary report",
     "CANNOT_RECALIBRATE": "Calibration cannot be re-run for this model fit please re-run fit step.",
-    "INVALID_ART_TIME_PERIOD": "Quarterly data not provided for all disaggregates or duplicate annual data provided for all disaggregates."
+    "INVALID_ART_TIME_PERIOD": "Quarterly data not provided for all disaggregates or duplicate annual data provided for all disaggregates.",
+    "ART_TOTAL": "ART count",
+    "ART_TOTAL_DESC": "Number on ART at the end of calendar year",
+    "ART_ADULT_F": "ART female",
+    "ART_ADULT_F_DESC": "Number of women (15+) on ART at the end of calendar year",
+    "ART_ADULT_M": "ART male",
+    "ART_ADULT_M_DESC": "Number of men (15+) on ART at the end of calendar year",
+    "ART_ADULT": "ART adult",
+    "ART_ADULT_DESC": "Number of adults (15+) on ART at the end of calendar year",
+    "ART_CHILD": "ART paediatric",
+    "ART_CHILD_DESC": "Number of children (<15) on ART at the end of calendar year",
```

Comment:
Looks like we've got many more keys added here than in the fr and pt translations - what is our preferred workflow here? To add dummy keys, google translations, or leave as-is? I think that as-is the key will get reflected out into the application, which is fine for now. We could also perhaps more easily detect missing keys by looking to see what is present in the en set but missing from the others?

Reply:
Plan is at some point before estimates round (probably some time early Nov) do a diff from the last date we received translations to now. Then add those new keys to a spreadsheet for UNAIDS to translate.

Have previously to this point been adding translations from google or deepl so it is at least partially usable by people but after discussion the other day maybe this is not worth the effort as it is quite boring to add these placeholder translations.